tests/test_integration.py

```diff
@@ -164,7 +164,7 @@ def do_GET(self):
                         assert req_id == "req_1"
                         # The first request took 31 ms, so the client perceived
                         # latency shouldn't be outside this range.
-                        assert 30 < duration_ms < 300
+                        assert 30 < duration_ms < 1000
```

Comment:
I increased the tolerance here because the test was occasionally failing with PyPy3.